lib/tasks/dev/populate.rake

```diff
@@ -2,18 +2,23 @@ require 'faker'
 require 'factory_bot_rails'
 
 namespace :dev do
-  desc 'populate dev database with sample student data'
+  desc 'populate dev database with sample student, course and institute data'
   task :populate => :environment do
     Rake::Task['db:reset'].invoke
     # Title table
     %w(Mr Mrs Miss Ms Dr Master).each_with_index { |v, i| 
         FactoryBot.create(:title, id: (i + 1), name: "#{v}")
-    } 
+    }
     # Student table
-    FactoryBot.create_list(:student, 50, title_id: rand(1..6))  
+    20.times do
+      FactoryBot.create(:student, title_id: rand(1..6))
+    end
+    
     # Course table
-    FactoryBot.create_list(:course, 20)
+    20.times do 
+      FactoryBot.create(:course, institute_id: rand(1..10))
```

Comment:
If you leave the institute field blank it will automatically create an institute for you

```suggestion
      FactoryBot.create(:course)
```

Reply:
That would be preferred over assigning random integers to the relation field

Reply:
Not sure if this is standard practice but atm I'm assigning the random institute_id so that it populates courses and students with different institutes and titles respectively. Is there a better way that you could recommend?

Reply:
Yea the suggested change above will create a course that has a institute (because of our factorybot definition). Alternatively you could create a institute and pass that into a course

```ruby
instutute = FactoryBot.create(:institute)
course = FactoryBot.create(:course, institute: institute)
```

There should be no need to mess with passing in IDs here